nextflow.config

```diff
@@ -17,7 +17,6 @@ params {
     fasta_dict                 = null
     fasta_mapperindexdir       = null
     fasta_circular_target      = null
-    fasta_mitochondrion_header = null
```

Comment:
if we still allow single genome input, does it make sense to remove this?

Reply:
There's already a params.mitochondrion_header under mt/nuclear ratio, so I considered this a duplicate